src/common/utils/encrypt.ts

```diff
@@ -0,0 +1,18 @@
+import { createCipheriv, createDecipheriv, scrypt, scryptSync } from 'crypto';
+
+export function encrypt(value: string, password: string) {
+  const key = scryptSync(password, 'salt', 32);
+  const cipher = createCipheriv('aes-256-ctr', key, Buffer.alloc(16, 0));
```

Comment:
You could have this "Buffer.alloc(16, 0" stored in a constant, and then reuse it.

Also, you should add line breaks between operations for better code readability especially before the return statement.